src/Games/NexusMods.Games.MountAndBlade2Bannerlord/MountAndBlade2BannerlordSettings.cs

```diff
@@ -0,0 +1,29 @@
+using NexusMods.Abstractions.Settings;
+
+namespace NexusMods.Games.MountAndBlade2Bannerlord;
+
+public class MountAndBlade2BannerlordSettings : ISettings
+{
+    public bool DoFullGameBackup { get; set; } = false;
+    public bool BetaSorting { get; set; } = false;
+
+
+    public static ISettingsBuilder Configure(ISettingsBuilder settingsBuilder)
+    {
+        return settingsBuilder.AddToUI<MountAndBlade2BannerlordSettings>(builder => builder
+            .AddPropertyToUI(x => x.DoFullGameBackup, propertyBuilder => propertyBuilder
+                .AddToSection(Sections.Experimental)
+                .WithDisplayName($"Full game backup: {MountAndBlade2Bannerlord.DisplayName}") 
+                .WithDescription("Backup all game folders, this will greatly increase disk space usage. Should only be changed before managing the game.")
+                .UseBooleanContainer()
+            )
+            .AddPropertyToUI(x => x.BetaSorting, propertyBuilder => propertyBuilder
+                .AddToSection(Sections.Advanced)
+                .WithDisplayName($"Beta Sorting: {MountAndBlade2Bannerlord.DisplayName}") 
```

Comment:
This setting needs a description that even players new to Bannerlord would be able to understand.

Even as a developer, without doing a deep dive, it's unclear whether 'Beta' here for example refers to 'Beta' in `Bannerlord.LauncherManager`, or whether it refers to a `Beta` version/feature of the game.

Reply:
Good point!
I guess we can actually remove the setting, since I don't believe the Beta sort is used a lot. But yea, it's a LauncherManager feature

Reply:
By the way feel free to resolve/collapse the segments where the code was changed.